libr/core/cmd_type.c

```diff
@@ -46,49 +46,8 @@ static int sdbdelete(void *p, const char *k, const char *v) {
 	return 1;
 }
 static int typelist(void *p, const char *k, const char *v) {
-#define DB core->anal->sdb_types
 	RCore *core = (RCore *)p;
-	int i;
```

Comment:
do not remove this code, wrap it with #if 0 .. #endif 
it can be useful at some point.